src/trampoline.js

```diff
@@ -0,0 +1,46 @@
+import isFunction from './isFunction';
+
+/**
+ * A function that repeatedly calls the return value of a function,
+ *  until it is no longer a function.
+ *
+ * @func trampoline
+ * @memberOf RA
+ * @category Function
+ * @sig ... -> Any
+ * @param  {function} func
+ * @param  {} ...args
+ * @return {*} Exclude<any, Function>
+ * @example
+ *
+ * function evenOline(n) {
```

Comment:
Can we make this code example compatible with our linting/formatting rules and prefer arrow functions instead of function declaration?